docs/changehistory/Roadmap.md

```diff
@@ -21,123 +21,129 @@ Volunteers for help on any or all of these projects are welcome. In particular,
+- [X] Add support for display of raster map layers connected to live WMS, WMTS, AzureMaps, MapBox, or ArcGIS REST API servers. Map layer visibility and transparency can be controlled independently. Layers can be attached either as background below BIM geometry or overlays above BIM geometry.
 
 ### Microsoft Teams integration
 
-- Provide api for hosting iTwin.js applications directly within Microsoft Teams.
+- [ ] Provide api for hosting iTwin.js applications directly within Microsoft Teams.
 
 ### ET/IT/OT Integration
 
-- Provide working examples and templates for interfacing between engineering content in iModels, enterprise systems, and realtime IOT sensors, cameras, controllers, devices, and processors.
-- Create tools for augmenting data synchronized with connectors from engineering design tools with IOT-link elements.
-- Enhance BIS schemas for common IOT patterns and query engines.
+- [ ] Provide working examples and templates for interfacing between engineering content in iModels, enterprise systems, and realtime IOT sensors, cameras, controllers, devices, and processors.
+- [ ] Create tools for augmenting data synchronized with connectors from engineering design tools with IOT-link elements.
+- [ ] Enhance BIS schemas for common IOT patterns and query engines.
 
 ### Agent Deployment
 
-- Provide working examples and templates of agents deployed via Kubernetes.
-- Perform operations on iModels that are triggered by relevant events from iModelHub.
+- [ ] Provide working examples and templates of agents deployed via Kubernetes.
+- [ ] Perform operations on iModels that are triggered by relevant events from iModelHub.
 
 ### iTwin Viewer
 
-- Create fully functional iTwin web viewing application platform, deployable on any cloud infrastructure. The Bentley iTwin Design Review product will be built from this application.
-- Provide necessary infrastructure to host Extensions within any iTwin Viewer-based solution.
+- [ ] Create fully functional iTwin web viewing application platform, deployable on any cloud infrastructure. The Bentley iTwin Design Review product will be built from this application.
+- [ ] Provide necessary infrastructure to host Extensions within any iTwin Viewer-based solution.
 
 ### Point Clouds
 
-- Support visualization of time-based differencing of large scale point clouds.
+- [ ] Support visualization of time-based differencing of large scale point clouds.
 
 ### Reality Data Sources
 
-- Implement a grid of linear transformations for large reality data.
+- [ ] Implement a grid of linear transformations for large reality data.
 
 ### Geolocated Photos and Panoramas
 
-- Improve server-side extraction of tags from photos and panoramas.
-- Incorporate geophoto package into an Extension loadable from any iTwin Viewer application.
+- [ ] Improve server-side extraction of tags from photos and panoramas.
```

Comment:
@calebmshafer Barry wrote this extension a long time ago, I think it's since been deleted. @Ellord207 do you have a replacement with your Earthcam stuff?

Reply:
iirc, orbitgt was supposed to replace barry's geo photo extension

Reply:
I'm not sure I understand - OrbitGT is point clouds, not photos - no?

Reply:
ah, i hope i didnt get that mixed up. 

if im not mistaken, @MarcBedard8 's reality data widget has an entrypoint for the orbitgt extension, and it loads geo located photos in a split content pane

Reply:
I think they extended it to do both to replace geo-photo. Not sure the plans to make this open source, if there are any. We'll need to follow up.

Reply:
I haven't used it, but IIRC OrbitGT had their own geo photo service (in addition to the point cloud stuff). It sounds like what Arun is describing is that having been integrated.

Reply:
@mgooding is right.  @jason-crow do you know if that's still in the works?

Reply:
we do have the orbitgt version integrated and it's my understanding that we can discard the old geophoto extension in favor of that, but currently we have both in DR @jjbeckman13 that's your understanding as well right? That with the 3.0 integration we can officially remove support of the old geophoto extension?

Reply:
Yes, that is my understanding as well: OrbitGt would replace GeoPhoto.